Mathlib/Tactic/Ring/Basic.lean

```diff
@@ -1086,6 +1086,19 @@ partial def eval {u} {α : Q(Type u)} (sα : Q(CommSemiring $α))
     | _ => els
   | _, _, _ => els
 
+class CSLift (α : Type u) (β : outParam (Type u)) where
+  lift : α → β
+  inj : Function.Injective lift
```

Comment:
Could we have a doc-string? At least telling me what `CS` stands for. :-)

Reply:
it's supposed to mean commutative semiring, although as you can see the class itself doesn't depend on it. I don't really have good naming suggestions here...